src/microbe_stage/editor/EndosymbiosisPopup.cs

```diff
@@ -141,6 +143,8 @@ private void ShowDataToStartNew(IEnumerable<KeyValuePair<Species, int>> candidat
 
             var choice = candidateGUIScene.Instantiate<EndosymbiosisCandidateOption>();
 
+            choice.Lawk = Lawk;
+
             MicrobeInternalCalculations.CalculatePossibleEndosymbiontsFromSpecies(microbeSpecies, tempSymbionts);
```

Comment:
I think a better implementation would be to not pass the LAWK flag to [EndosymbiosisCandidateOption.cs](https://github.com/Revolutionary-Games/Thrive/pull/5319/files#diff-1f85d8238700eb6c64d6bc917a83b6cce349be676eaa3a4ab18e8deae5bef464) and instead filter inside the CalculatePossibleENdosymbiontsFromSpecies method.